benchmarks/benchmark_throughput.py

```diff
@@ -515,7 +524,7 @@ def validate_args(args):
     parser.add_argument(
         "--dataset-name",
         type=str,
-        choices=["sharegpt", "random", "sonnet", "burstgpt", "hf"],
+        choices=["sharegpt", "random", "sonnet", "burstgpt", "hf", "instructcoder"],
```

Comment:
I think we can keep the instructcoder dataset name as hf, but the data path should be likaixin/InstructCoder, just like the vision arena.

Reply:
yup, updated